modules/ROOT/pages/configure-adv-tls-context.adoc

```diff
@@ -0,0 +1,81 @@
+= Import a TLS Context from Secrets Manager (Advanced)
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If your environment has specific security requirements or requires advanced configuration, you can import a TLS context from the secrets manager.
+
+== Prerequisites
+
+You must have the following permissions to enable inbound traffic to Runtime Fabric:
```

Comment:
Given the change in topic, we should consider rewording this to something like "to import a TLS context from secrets manager"

Reply:
Good catch - I updated the sentence.